### PR TITLE
fix: onboarding fixes

### PR DESCRIPTION
**Pull request by MounirDhahri**
The type of this PR is: **Bugfix**

Addresses [CX-1903]

This PR resolves:
- Palette v3 issues with onboarding
- Onboarding copy

### Description

- Add missing copy that we agreed on here 
https://www.notion.so/artsy/Onboarding-Socail-auth-error-messages-9aeb58a81f8042a185d55effc27cb461

- Fix Following UI Issues:
**Before**
Group 4

**After**
Group 3 (1)

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Palette v3 onboarding issues - mounir
- Onboarding copy error message - mounir

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

[CX-1903]: https://artsyproduct.atlassian.net/browse/CX-1903

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (Palette v3 onboarding issues - mounir,Onboarding copy error message - mounir)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against ea4095fe33f5f5eb597797caa675dbc217ee781f
</p>

**Review by olerichter00 — APPROVED** [review]
Amazing! Thanks a lot for the fix!

Although some specs are failing because of some issues with the font size.

File: src/lib/Scenes/Onboarding/OnboardingWelcome.tsx [hunk 1]
```diff
@@ -121,14 +121,14 @@ export const OnboardingWelcome: React.FC<OnboardingWelcomeProps> = ({ navigation
 
       <Animated.View style={{ flex: 1, padding: space(2), justifyContent: "flex-end", opacity }}>
         {/* <Flex flex={1} p={2} justifyContent="flex-end"> */}
```
[the remark this answers is not in the document]
Comment (MounirDhahri) [reply src/lib/Scenes/Onboarding/OnboardingWelcome.tsx:123]: oh yes sure

File: src/lib/Scenes/Onboarding/OnboardingWelcome.tsx [hunk 2]
```diff
@@ -121,14 +121,14 @@ export const OnboardingWelcome: React.FC<OnboardingWelcomeProps> = ({ navigation
...
-        <Text color="white" fontSize="48px" lineHeight={48}>
+        <TextV2 color="white" fontSize="48px" lineHeight={48}>
```
[the remark this answers is not in the document]
Comment (MounirDhahri) [reply src/lib/Scenes/Onboarding/OnboardingWelcome.tsx:124]: Actually I just realized there might be a way and that will fix the same issue in my areas of the app. I will push it now

[the remark this answers is not in the document]
Comment (olerichter00) [reply src/lib/Scenes/Onboarding/OnboardingWelcome.tsx:124]: Yeah!

**Comment by olerichter00** [comment]
Ready to be merged after fixing the tests (I think it's just the copy that needs to be changed)

**Comment by MounirDhahri** [comment]
#squashongreen

**Pull request merged**